src/overlays/actors/ovl_Boss_Goma/z_boss_goma.c

```diff
@@ -1847,7 +1847,7 @@ void BossGoma_UpdateHit(BossGoma* this, PlayState* play) {
                 }
 
                 this->timer = 4;
-                func_80033E88(&this->actor, play, 4, 0xC);
+                Actor_RequestQuakeAndRumble(&this->actor, play, 4, 0xC);
```

Comment:
another easier comment
```suggestion
                Actor_RequestQuakeAndRumble(&this->actor, play, 4, 12);
```